cfgov/unprocessed/apps/regulations3k/js/permalinks-utils.js

```diff
@@ -1,5 +1,13 @@
 // Array that tracks paragraph positions
 let paragraphPositions;
+const wayfinderOffset = 45;
```

Comment:
Per @niqjohnson's comment above, maybe calculate this dynamically?

```suggestion
const wayfinderOffset = regs3kWayfinder.scrollHeight + 10;
regs3kWayfinder.style.top = -Math.abs(wayfinderOffset) + 'px';
```